### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by darylknight** [ad-dc/focalpoint_migration#1]
Hey - thanks for this!

Is this meant to be run before or after the upgrade to Craft 3?
Which table is meant to go in `$asset_table_name = "";` ? assetfiles?

**Comment by CreateSean** [comment]
I used "assets" and it worked for me.

**Comment by darylknight** [comment]
I'm just seeing `Connected` in the log, then nothing else.

**Comment by CreateSean** [comment]
Daryl, on Windows you need to include the full path to php.exe, php.ini and then migrate.php

**Comment by darylknight** [comment]
Thanks Sean! I'm back on MacOS now. It's half working as it does get connected, it just doesn't seem to do anything after it's connected.

**Comment by matthillco** [comment]
I was having the same problem as Daryl. Looked at the code and found the error: In `settings.php`, there's a variable for `$db_prefix` but it's never used. So just make sure that you add your table prefix in the relevant table identifiers instead, otherwise the script won't find the tables and will do nothing:

```
$fp_table_name = "craft_focuspoint_focuspoints";
$asset_table_name = "craft_assets";
```

**Comment by autumn-hoerr** [comment]
Should be fixed by ea9ead3efe43fb003ed0157fd78be32252b1d11a
